core/util.go

```diff
@@ -422,3 +423,11 @@ func RetryBackoff(retries int, base, max time.Duration, factor float64) time.Dur
 	backoff *= (1 - retryJitter) + 2*retryJitter*mrand.Float64()
 	return time.Duration(backoff)
 }
+
+// LowerAndSort lowercases each string in a slice before sorting it
+func LowerAndSort(strs []string) {
```

Comment:
These should also be uniqued, right?

Maybe we just fix the UniqueLower func here to also do sorting? That'll close another ticket we have open, too.

I think maybe this doesn't even need to be here, anyway, since only one package should really be worrying about it.